installer/Migration/V5_1_0/Migration.php

```diff
@@ -37,6 +42,41 @@ public function up(): void
             ['onCascade' => 'DELETE']
         );
         $this->getSchemaHelper()->addForeignKey('ohrm_kpi', $kpiForeignKeyConstraint);
+
+        $groups = ['recruitment'];
+        foreach ($groups as $group) {
+            $this->getLangStringHelper()->deleteNonCustomizedLangStrings($group);
+            $this->getLangStringHelper()->insertOrUpdateLangStrings($group);
+        }
+
+        $langCodes = [
+            'en_US',
```

Comment:
Aren't we adding the bg_BG translations? 
What about other languages?

Reply:
yes we have